examples/gallery/symbols/basic_symbols.py

```diff
@@ -0,0 +1,117 @@
+"""
+Basic geometric symbols
+-----------------------
+
+The :meth:`pygmt.Figure.plot` method can plot individual geometric symbols
+by passing the corresponding shortcuts to the ``style`` parameter. The 14 basic
+geometric symbols are shown underneath their corresponding shortcut codes.
+"""
+
+import pygmt
+
+fig = pygmt.Figure()
+fig.basemap(region=[0, 8, 0, 3], projection="X12c/4c", frame=True)
+
+# define fontstlye for annotations
+font = "15p,Helvetica-Bold"
+
+# upper row
+y = 2
+
+# use a dash in x direction (-) with a size of 0.9 cm,
+# linewidth is set to 2p and the linecolor to "gray40"
+x = 1
+fig.plot(x=x, y=y, style="-0.9c", pen="2p,gray40")
+fig.text(x=x, y=y + 0.6, text="-", font=font)
+
+# use a plus (+) with a size of 0.9,
+# linewidth is set to 2p and the linecolor to "gray40"
+x += 1
+fig.plot(x=x, y=y, style="+0.9c", pen="2p,gray40")
+fig.text(x=x, y=y + 0.6, text="+", font=font)
+
+# use a star (a) with a size of 0.9,
+# linewidth is set to 1p, the linecolor to "black" (default) and the
+# color fill to "darkorange"
+x += 1
+fig.plot(x=3, y=y, style="a0.9c", pen="1p,black", color="darkorange")
```

Comment:
Should you use `x=x` here?
```suggestion
fig.plot(x=x, y=y, style="a0.9c", pen="1p,black", color="darkorange")
```

Reply:
Or we can use `x=x+1`, `x=x+2` etc? 
```suggestion
fig.plot(x=x+2, y=y, style="a0.9c", pen="1p,black", color="darkorange")
```

Reply:
Actually, I don't mind using numbers `x=1`, `x=2`, `x=3` which may make the code even easier to read for beginners.

Reply:
> Actually, I don't mind using numbers `x=1`, `x=2`, `x=3` which may make the code even easier to read for beginners.

Sounds good~

Reply:
Adjusted it to use numbers ``x=1``, ``x=2``, ``x=3`` and so on.